oss_src/fileio/fileio_constants.cpp

```diff
@@ -162,6 +164,10 @@ const std::string& get_alternative_ssl_cert_file() {
   return FILEIO_ALTERNATIVE_SSL_CERT_FILE;
 }
 
+const bool disable_ssl_cert_checks() {
+  return FILEIO_DISABLE_SSL_CERTIFICATE_CHECKS != 0;
+}
```

Comment:
eh...does this tell you if ssl cert checks are disabled? Weird name for it, if so.